### ISSUE TITLE
Reduce validation loss in training

### ISSUE THREAD

**Issue by egpbos** [spokenlanguage/platalea#30]
In run [dainty-dawn-20](https://wandb.ai/spokenlanguage/platalea_transformer/runs/31ivn5al) we saw the validation loss increasing again starting from epoch 20, approximately. We should find a way to train that reduces validation loss together with training loss.

**Comment by egpbos** [comment]
First try: dropout = 0.1. Run: [lunar-serenity-26](https://wandb.ai/spokenlanguage/platalea_transformer/runs/2in7wtbv).

**Comment by egpbos** [comment]
Promising, surely better results than with 0 dropout. Now trying `dropout = 0.2`: [dark-serenity-27](https://wandb.ai/spokenlanguage/platalea_transformer/runs/2lpdqvnb). Note: this is only dropout in the transformer encoder layer.

**Comment by egpbos** [comment]
Also tried 0.5 (worthy-sunset-28). Both cases: difference between validation loss and step loss stays slightly lower initially than in the lower dropout factor runs.

However, also running [logical-butterfly-29](https://wandb.ai/spokenlanguage/platalea_transformer/runs/2qvp6hf2) and there we see that after some more training the validation loss still starts climbing again, whereas the training loss more or less completely vanishes, so again overfit on training set at some point. The performance is better though, and still climbing at this moment (epoch 130 of 300), so let's see where it goes.

**Comment by egpbos** [comment]
We did a [sweep again](https://wandb.ai/spokenlanguage/platalea_transformer/sweeps/1esvjk7f?workspace=user-egpbos) over dropout values and again values between 0.2-0.5 seem to perform best. However, validation loss still increases in all runs after a certain time.

As already mentioned in [this report](https://wandb.ai/spokenlanguage/platalea_transformer/reports/Report-of-the-learning-rate-sweep-1--Vmlldzo0MDA2NjM), we should probably look into more regularization options to correct for this overfitting on training data. See #61.

**Comment by cwmeijer** [comment]
I think we solved the validation loss (overfitting) issue during our regularization sweeps (https://wandb.ai/spokenlanguage/platalea_transformer/reports/Jan-29-Project-Update-Regularization-rates-conclusion--Vmlldzo0MzY3MDg).
